.buildkite/pipeline.yml

```diff
@@ -11,6 +11,21 @@ env:
   TERRAFORM_VERSION: "1.6.4"
 
 steps:
+
+  - label: "Validate catalog-info"
+    if: build.pull_request.id != null
+    plugins:
+      monorepo-diff#v1.3.0:
+        diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+        wait: true
+        watch:
+          # if our catalog-info.yaml is changed, make sure it's well-formed according to our internal standards as well as Backstage's validation
+          - path: "catalog-info.yaml"
+            config:
+              command: "/agent/check-catalog-info.sh"
+              agents:
+                image: "docker.elastic.co/ci-agent-images/pipelib:0.13.0@sha256:190464950de81cb3390bd5e07bb1cc171dd4a765c1ce5de58ebca476f9a6839f"
```

Comment:
Just wondering if we need the specific version tag + SHA here.  Couldn't we always pull the latest as mentioned in https://github.com/elastic/ci-agent-images/tree/main/container-images/engprod/pipelib#validating-your-renovate-configuration-in-a-pipeline?

Reply:
I think that's related to some security practice where dependencies are not using mutable tags but fixed versions with the sha256.

I took the configuration from the Onboarding templates that the platform eng prod team provides.

Let me ping you in the private GH PR